### ISSUE TITLE
No Tears Guide to Gaming is a Tutorial, not a Case Study

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#420]
```
Need to move the article around a bit.
```

Original issue reported on code.google.com by `sethladd@google.com` on 5 Feb 2011 at 12:21

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5a4a31be8e.
```

Original comment by `ericbide...@html5rocks.com` on 8 Feb 2011 at 7:37
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Commit 
http://code.google.com/p/html5rocks/source/detail?r=5a4a31be8efb0bb03b494caf2e22
d04b5537e5c3
```

Original comment by `ericbide...@html5rocks.com` on 8 Feb 2011 at 7:38
- Added labels: **Milestone-5.2**, **Tutorial**
